src/patternfly/components/TabContent/tab-content.hbs

```diff
@@ -0,0 +1,11 @@
+<section class="pf-c-tab-content{{#if tab-content--modifier}} {{tab-content--modifier}}{{/if}}"
```

Comment:
Is the tab content meant to be focusable? When I click on it, it seems editable.

Reply:
Yep, that's to make the content area focusable per [Jenn's comment](https://github.com/patternfly/patternfly-next/pull/1371#issuecomment-461030846) referencing https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html